src/lib/system.ts

```diff
@@ -0,0 +1,21 @@
+export enum Mode {
+    CLOUD = 'cloud',
+    SELF_HOSTED = 'self-hosted'
+}
+
+export const VARS = {
+    APPWRITE_ENDPOINT: import.meta.env?.VITE_APPWRITE_ENDPOINT?.toString() as string | null,
```

Comment:
Why do you need the cast? Also, shouldn't it be `string | undefined`?

Reply:
![Bildschirm­foto 2023-02-14 um 17 56 25](https://user-images.githubusercontent.com/1759475/218804970-4388dc2f-d6be-4bc8-bf93-a28af20e187d.png)

thanks for the catch, the types are otherwise any 🙃